src/plays/password-generator/PasswordGenerator.jsx

```diff
@@ -58,89 +72,147 @@ function PasswordGenerator(props) {
 
   // Generate a random password
   const generatePassword = () => {
-    setError(false);
-    const concated = arrangeData();
-    let finalPassword = "";
-    for (let i = 0; i < passwordConfig.length; i++) {
-      finalPassword += concated[randomNumberGenerator(concated.length)];
-    }
-    return finalPassword;
+
+      setError(false);
+
```

Comment:
There are lots of line breaks, lets avoid that,

Reply:
line breaks and responsiveness 
Let me clean it up now and send another PR

Reply:
I have cleaned the code and set up responsiveness